cart/domain/cart/cart.go

```diff
@@ -276,7 +275,7 @@ func (c Cart) GetDeliveryByItemID(itemID string) (*Delivery, error) {
 		}
 	}
 
-	return nil, errors.Errorf("delivery not found for %q", itemID)
+	return nil, fmt.Errorf("delivery not found for %q", itemID) //nolint:err113 //not worth introducing a dedicated error
```

Comment:
looks to my like not found with different comments, why not worth?

Reply:
valid point, updated the code to reuse existing errors instead of muting the linter..